File: C/src/1009.c
#include <stdio.h>

int main(){
    char str[20];
    float salary,selling;
    scanf("%s\n%f\n%f",&str,&salary,&selling);
    float TotalToPay = salary + (selling*15)/100;
    printf("TOTAL = R$ %.2f\n",TotalToPay);
    return 0;
}

File: go/1009.go
package main

import "fmt"

func main() {
	var name string
	var sallary, selling float64
	fmt.Scanln(&name)
	fmt.Scanln(&sallary)
	fmt.Scanln(&selling)
	sallary += selling * 0.15
	fmt.Printf("TOTAL = R$ %.2f\n", sallary)
}
